trailblazer/store/crud/update.py

```diff
@@ -43,3 +54,21 @@ def update_analysis_jobs_from_slurm_jobs(
             for job in squeue_result.jobs
         ]
         self.commit()
+
+    def _update_analysis_status_from_slurm_jobs(self, analysis: Analysis) -> None:
+        """Update analysis status based on current SLURM jobs status."""
+        squeue_result: SqueueResult = get_squeue_result(
+            squeue_response=get_slurm_squeue_output(slurm_job_id_file=Path(analysis.config_path))
+        )
+        self.update_analysis_jobs_from_slurm_jobs(analysis=analysis, squeue_result=squeue_result)
+        LOG.info(f"Status in SLURM: {analysis.family} - {analysis.id}")
```

Comment:
Is this really needed when we log the new status below?

Reply:
Changed to debug.